--- rdiff-backup-1.2.0/setup.py~	2008-07-27 15:57:49.000000000 +0200
+++ rdiff-backup-1.2.0/setup.py	2008-07-31 00:02:01.000000000 +0200
@@ -74,8 +74,6 @@
 							   extra_link_args=lflags_arg)],
 	  scripts = ['rdiff-backup', 'rdiff-backup-statistics'],
 	  data_files = [('share/man/man1', ['rdiff-backup.1',
-										'rdiff-backup-statistics.1']),
-					('share/doc/rdiff-backup-%s' % (version_string,),
-					 ['CHANGELOG', 'COPYING', 'README', 'FAQ.html'])],
+										'rdiff-backup-statistics.1'])],
 					**extra_options)
 
